docs/LOCAL_DEPLOYMENT.md

```diff
@@ -1,52 +1,31 @@
-
-*npm install -g iot-solutions*
+1) *If you have cloned azure-iot-pcs-remote-monitoring-java repository, the scripts folder is present under services submodule (folder).*
+2) *This script requires **Node.js** to execute, please install Node (version < 8.11.2) before using this script. Also, this script might require administartive privileges or sudo permission as it tries to install [pcs-cli](https://github.com/Azure/pcs-cli) a cli interface for remote-monitoring deployments.*
 &nbsp; 
 
-##### Usage:   
-````
-abc@pcs sh start.sh   
-````
-![start](https://user-images.githubusercontent.com/39531904/44435771-6ab08280-a566-11e8-93c9-e6f35e5df247.PNG)
- 
-#### Helper scripts
-These scripts are located under helpers folder which is under the launch folder. The script create_azure_resources.sh can be independently called to create resources in the cloud. The script check_dependencies.sh checks if environment variables are set for a particular microservices.
-##### Usage:
-1) check environment variables for a microservice 
-sh check_dependencies.sh <microservice_folder_name> 
-2) create Azure resources 
-sh create_azure_resources.sh
- 
-After creating the required azure resources, using start or create-azure-resources.sh, one should execute the following scripts present under *os/{linux / win / osx}* to set the environment variables. 
-1) set-env-uri
-2) set-env
+#### Existing Users
+For users who have already created the required azure resources, please **set the environment variables** 
+1) Globally on your machine so as to be accessible by the IDE. OR
+2) In "Edit configurations" vizard of the Intellji IDE. 
 
-**Please Note:**
-*If you are using windows, you will have to execute these scripts in CMD shell. On OSX, these scripts are automatically run by the start script. For linux, the environment variables present in these scripts need to be set at global location, depending upon the flavour of linux you are using.* 
+*Although not recommended, environment variables can also be set in application.conf file present under webservice package for each of the microservices.*
 
-#### Recap of steps to create resources and set environment variables
-1) Run start.sh
-2) Run scripts under os folder. 
 
-#### Walk through for importing new Solution in IDE
+### Walk through for importing new Solution in IDE
 ##### Intellji
 This is our preferred editor for Java development.
 
-The java repository has a similar structure to the dotnet repo and contains the same script(s). The scripts, described above, are located under scripts/local/launch folder.  
+The java repository has a similar structure to the java repo and contains the same script(s). The scripts, described above, are located under scripts/local/launch folder.  
```

Comment:
undo java to dotnet. Or may be no need to call out that they are similar.

Reply:
ok